solana-programs/anchor/audius-data/tests/audius-data.ts

```diff
@@ -882,18 +710,95 @@ describe("audius-data", function () {
       updateUser({
         program,
         metadata: randomCID(),
-        userStorageAccount: newUserAcctPDA,
-        userAuthorityKeypair: userAuthorityDelegateKeypair,
-        userAuthorityDelegate: userAuthorityDelegatePDA,
-        authorityDelegationStatusAccount: authorityDelegationStatusPda,
+        userStorageAccount: delegate.userAccountPDA,
+        userAuthorityKeypair: delegate.userAuthorityDelegateKeypair,
+        userAuthorityDelegate: delegate.userAuthorityDelegatePDA,
+        authorityDelegationStatusAccount: delegate.authorityDelegationStatusPDA,
       })
     )
       .to.eventually.be.rejected.and.property("msg")
       .to.include(`This authority's delegation status is revoked.`);
   });
 
+  it("delegate adds/removes another delegate", async function () {
+    const firstDelegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    const secondDelegate = await testCreateUserDelegate({
+      adminKeypair,
+      adminStorageKeypair,
+      program,
+      provider,
+      ...getURSMParams(),
+    });
+
+    const userAuthorityDelegateSeeds = [
+      firstDelegate.userAccountPDA.toBytes().slice(0, 32),
```

Comment:
i dunno if they should be called `firstDelegate` and `secondDelegate` here - it makes it unclear why we use `firstDelegate.userAccountPDA` when actually the `userAccountPDA` is the same for both, for instance.